mongodbatlas/resource_mongodbatlas_serverless_instance.go

```diff
@@ -259,11 +259,11 @@ func resourceMongoDBAtlasServerlessInstanceRead(ctx context.Context, d *schema.R
 	if err := d.Set("connection_strings_standard_srv", serverlessInstance.ConnectionStrings.StandardSrv); err != nil {
 		return diag.Errorf("error setting `connection_strings_standard_srv` for serverless instance (%s): %s", d.Id(), err)
 	}
-	if len(serverlessInstance.ConnectionStrings.PrivateEndpoint) > 0 {
```

Comment:
This `if` was the bug as we want to populate the `ConnectionStrings.PrivateEndpoint` with an empty array